build/build.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/urfave/cli"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var packages = []string{"cli", "altsrc"}
+
+func main() {
+	app := cli.NewApp()
+
+	app.Name = "builder"
+	app.Usage = "Generates a new urfave/cli build!"
+
+	app.Commands = cli.Commands{
+		cli.Command{
+			Name:   "vet",
+			Action: VetActionFunc,
+		},
+		cli.Command{
+			Name:   "test",
+			Action: TestActionFunc,
+		},
+		cli.Command{
+			Name:   "gfmrun",
+			Action: GfmrunActionFunc,
+		},
+		cli.Command{
+			Name:   "toc",
+			Action: TocActionFunc,
+		},
+		cli.Command{
+			Name:   "generate",
+			Action: GenActionFunc,
+		},
+	}
+
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func VetActionFunc(_ *cli.Context) error {
+	return exec.Command("go", "vet").Run()
+}
+
+func TestActionFunc(c *cli.Context) error {
+	for _, pkg := range packages {
+		var packageName string
+
+		if pkg == "cli" {
+			packageName = fmt.Sprintf("github.com/urfave/cli")
```

Comment:
Could just be a string without the sprintf

Reply:
agree. will change this

Reply:
this is done